Ports/pacman/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=pacman
+version=git
+workdir=pacman-master
+files="https://github.com/ebuc99/pacman/archive/refs/heads/master.tar.gz pacman-master.tar.gz 3016af69e168e0dd87a079bbfb35d995cf2e7bc1b710716c81ac99323bb4f7d5"
+auth_type=sha256
+useconfigure=true
+depends=("SDL2" "SDL2_image" "SDL2_ttf" "SDL2_mixer")
+configopts=(
+    "--with-sdl-prefix=${SERENITY_INSTALL_ROOT}/usr/local"
+)
+launcher_name="Pacman"
+launcher_category="Games"
+launcher_command=/usr/local/bin/pacman
+icon_file="data/gfx/pacman_desktop.png"
+
+pre_patch() {
+    run ./autogen.sh
```

Comment:
This seems like it replaces the `config.sub` in the repository (as the repository itself only features a `config.sub` from 2014, which lacks support for `serenity`). This ultimately makes port buildability depend on the version of autotools that is installed on the host system.

Please enable `use_fresh_config_sub` and point `config_sub_paths` to the correct file to ensure that we always have a sufficiently new `config.sub` file.